WalletWasabi.Fluent/ViewModels/MainViewModel.cs

```diff
@@ -328,4 +334,14 @@ private SearchBarViewModel CreateSearchBar()
 
 		return searchBar;
 	}
+
+	private void InitializeSingleton()
+	{
+		if (Instance != null)
+		{
+			throw new InvalidOperationException($"MainViewModel instantiated more than once.");
+		}
+
+		Instance = this;
+	}
```

Comment:
@ichthus1604 I was wrong here, since `Instance` is a static property, even if you set it from ctor it will give you a nullable warning. Because it can be null until someone initialize a MainViewModel. 

It seems to me that is why it was initialized there with `new()`.

Reply:
@soosr there are currently 19 usages of `MainViewModel.Instance` throughout the codebase. If I mark it as nullable, I'll have to add null checks to these 19 call sites.

I suggest to leave it as-is for now, since that property needs to go away anyways.

WDYT?

Reply:
If it will be removed, I am ok with it for now, make sense.